appcues/src/main/java/com/appcues/data/remote/response/step/StepResponse.kt

```diff
@@ -12,5 +12,5 @@ internal data class StepResponse(
     val content: PrimitiveResponse,
     val traits: List<TraitResponse>,
     val actions: Map<UUID, List<ActionResponse>>,
-    val type: String?,
+    val type: String,
```

Comment:
@andretortolano a head's up that my latest commit adds a unit test for this deserialization logic, and made this minor change to make `type` required here (and in the `StepOrContainerResponse` line 21 above). This is to match iOS and the spec https://github.com/appcues/appcues-mobile-experience-spec/blob/main/experience-schema.json#L64 for consistency.